src/apps/reservation-list/list/reservation-list.test.ts

```diff
@@ -885,10 +885,8 @@ describe("Reservation list", () => {
       "/iframe.html?path=/story/apps-reservation-list--reservation-list-entry"
     );
 
-    cy.interceptRest({
-      aliasName: "work-bestrepresentation",
-      httpMethod: "POST",
-      url: "**/next*/**",
+    cy.interceptGraphql({
+      operationName: "getManifestationViaBestRepresentationByFaust",
       fixtureFilePath: "reservation-list/work-bestrepresentation.json"
```

Comment:
Nice to have this fixed up. Before we intercept rest for a GraphQL call using a Rest interception. That it not right.